<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageWindow</class>
 <widget class="QMainWindow" name="MessageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>539</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralwidgetGLayout">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="toolBarFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="toolBarFrameGLayout">
       <property name="margin">
        <number>2</number>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="newmessageButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>New Message</string>
         </property>
         <property name="text">
          <string>Compose</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/folder-draft24.png</normaloff>:/images/folder-draft24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="rightVLine">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="replymessageButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Reply to selected message</string>
         </property>
         <property name="text">
          <string>Reply</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/replymail-pressed.png</normaloff>:/images/replymail-pressed.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="replyallmessageButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Reply all to selected message</string>
         </property>
         <property name="text">
          <string>Reply all</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/replymailall24-hover.png</normaloff>:/images/replymailall24-hover.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="forwardmessageButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777</width>
           <height>16777</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Forward selected message</string>
         </property>
         <property name="text">
          <string>Forward</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/mailforward24-hover.png</normaloff>:/images/mailforward24-hover.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="Line" name="leftVLine">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QToolButton" name="removemessageButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Remove selected message</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/deletemail24.png</normaloff>:/images/deletemail24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QToolButton" name="printbutton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Print selected message</string>
         </property>
         <property name="text">
          <string>Print</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/print24.png</normaloff>:/images/print24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QToolButton" name="viewtoolButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Display</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/looknfeel.png</normaloff>:/images/looknfeel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <spacer name="toolBarHSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="6">
        <widget class="QToolButton" name="tagButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Tags</string>
         </property>
         <property name="text">
          <string>Tags</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/tag24.png</normaloff>:/images/tag24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="msgLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionIconOnly">
   <property name="text">
    <string>Buttons Icon Only</string>
   </property>
   <property name="toolTip">
    <string>Buttons Icon Only</string>
   </property>
  </action>
  <action name="actionTextBesideIcon">
   <property name="text">
    <string>Buttons Text Beside Icon</string>
   </property>
   <property name="toolTip">
    <string>Buttons with Text</string>
   </property>
  </action>
  <action name="actionTextUnderIcon">
   <property name="text">
    <string>Buttons Text Under Icon</string>
   </property>
   <property name="toolTip">
    <string>Set Text Under Icon</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
